pages/manage_and_operate/cli/cli-getting-started/guide.en-gb.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Getting Started with OVHcloud CLI
+excerpt: Find out about useful resources in order to use the official OVHcloud Command Line Interface (CLI)
+updated: 2025-09-16
+---
+
+## Getting Started with OVHcloud CLI
+
+The OVHcloud Command Line Interface (CLI) is a tool designed to interact with OVHcloud services directly from your terminal. It allows you to automate tasks, manage resources, and access OVHcloud APIs efficiently.
+
+## Installation
+
+To install the OVHcloud CLI, you can use the following command:
+
+```sh
+curl -fsSL https://raw.githubusercontent.com/ovh/ovhcloud-cli/main/install.sh | sh
+```
+
+Alternatively, you can download the latest release from the [GitHub repository](https://github.com/ovh/ovhcloud-cli):
+
+1. Visit the [releases page](https://github.com/ovh/ovhcloud-cli/releases).
+2. Download the binary for your operating system (Linux, macOS, or Windows).
+3. Unpack the archive and move the binary to a directory included in your `$PATH`.
+4. Verify the installation:
+
+```sh
+ovhcloud version
+```
+
+## Authenticating the CLI
+
+OVHcloud CLI requires authentication to be able to make API calls. There are several ways to define your credentials.
+
+Check out the [authentication page](https://github.com/ovh/ovhcloud-cli/blob/main/doc/authentication.md) for further information about the configuration and the authentication means.
+
+* Using a configuration file:
+
+Default settings can be set using a configuration file named `.ovh.conf` and located in your `${HOME}` directory.
+
+Example of configuration file:
+
+```ini
+[default]
+endpoint = ovh-eu
+
+[ovh-eu]
+client_id     = <OAuth 2.0 client ID>
+client_secret = <OAuth 2.0 client secret>
```

Comment:
Switching to AppKey + AppSecret as no UI are available to generate OAuth2 client credentials.